ui-frontend/packages/catalog-ui-search/exports/sharing.tsx

```diff
@@ -12,8 +12,9 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  **/
+// @ts-nocheck ts-migrate(2307) FIXME: Cannot find module
```

Comment:
📓 The file that's being imported/exported here doesn't exist. Can we delete this file?